src/_pytest/runner.py

```diff
@@ -236,16 +236,9 @@ def from_call(cls, func, when, reraise=None):
         return cls(start=start, stop=stop, when=when, result=result, excinfo=excinfo)
 
     def __repr__(self):
-        if self.excinfo is not None:
-            status = "exception"
-            value = self.excinfo.value
-        else:
-            # TODO: investigate unification
-            value = repr(self._result)
-            status = "result"
-        return "<CallInfo when={when!r} {status}: {value}>".format(
-            when=self.when, value=value, status=status
-        )
+        if self.excinfo is None:
+            return "<CallInfo when={!r} result: {!r}>".format(self.when, self._result)
+        return "<CallInfo when={!r} exc: {!r}>".format(self.when, self.excinfo.value)
```

Comment:
based on the final traceback, how about using ""<CallInfo when={!r} {!r}: {}>".format(self.when, self.excinfo.type, self.excinfo.value)"

soit would spell `<CallInfo when='123' ZeroDivisionError: division by zero>

Reply:
Should use `str(self.excinfo.type.__name__)` then likely, and using `str(self.excinfo.value)` has the original issue, that it would contain literal "\n".

Reply:
I think just using `repr(self.excinfo)` is good - it is always good to use reprs in `__repr__` after all.
We could drop the "exc: " prefix though?

Reply:
@RonnyPfannschmidt 
> based on the final traceback

What do you mean here btw?

Reply:
Having `repr(self.excinfo)` there might be also good/better, since it would include `tblen`:
`<ExceptionInfo AssertionError('tearDown\nassert 0') tblen=1>`, but makes it a bit longer.
Having it as `excinfo=<ExceptionInfo AssertionError('tearDown\nassert 0') tblen=1>` therein would clearly indicate that it is a property, too.

Reply:
i like it

wrt the trace-back, i was starting at the trace back in the pr about that character difference in the  zerodivisionerror

Reply:
> i like it

Which one?  The whole repr of excinfo with `excinfo=` prefix?

Reply:
yup

Reply:
Amended.